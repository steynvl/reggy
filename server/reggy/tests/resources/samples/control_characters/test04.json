{
  "sampleStringsInfo": [
    [
      {
        "markerType": "Control characters",
        "markedStrings": [
          "das"
        ],
        "markerInfo": {
          "nul": false,
          "startOfHeading": false,
          "startOfText": false,
          "endOfText": false,
          "endOfTransmission": false,
          "enquiry": false,
          "acknowledge": false,
          "bell": true,
          "backspace": false,
          "horizontalTab": false,
          "newLine": true,
          "verticalTab": false,
          "formFeed": false,
          "carriageReturn": true,
          "shiftOut": false,
          "shiftIn": false,
          "dataLinkEscape": false,
          "deviceControlOne": false,
          "deviceControlTwo": false,
          "deviceControlThree": false,
          "deviceControlFour": true,
          "negativeAcknowledge": false,
          "synchronousIdle": false,
          "endOfTransBlock": false,
          "cancel": false,
          "endOfMedium": false,
          "substitute": false,
          "escape": false,
          "fileSeparator": false,
          "groupSeparator": false,
          "recordSeparator": false,
          "unitSeparator": false,
          "matchAllExceptSelectedOnes": false
        },
        "repeatInfo": {
          "repeat": "n or more times",
          "start": "7"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
