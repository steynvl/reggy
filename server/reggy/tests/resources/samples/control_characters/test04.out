Java
Pattern regex = Pattern.compile("[\\x07\\x0A\\x0D\\x14]{7,}");
[\\x07\\x0A\\x0D\\x14]{7,}
Perl
my $regex = /[\x07\x0A\x0D\x14]{7,}/;
[\x07\x0A\x0D\x14]{7,}
POSIX
[\x07\x0A\x0D\x14]{7,}
[\x07\x0A\x0D\x14]{7,}
Python
regex = re.compile(r'[\x07\x0A\x0D\x14]{7,}')
[\x07\x0A\x0D\x14]{7,}
JavaScript
const regex = /[\x07\x0A\x0D\x14]{7,}/;
[\x07\x0A\x0D\x14]{7,}
PHP
$regex = '/[\x07\x0A\x0D\x14]{7,}/';
[\x07\x0A\x0D\x14]{7,}
Golang
regex, _ := regexp.Compile("[\\x07\\x0A\\x0D\\x14]{7,}")
[\\x07\\x0A\\x0D\\x14]{7,}
Rust
let re = Regex::new(r"[\x07\x0A\x0D\x14]{7,}").unwrap();
[\x07\x0A\x0D\x14]{7,}
C#
var re = new Regex(@"[\x07\x0A\x0D\x14]{7,}");
[\x07\x0A\x0D\x14]{7,}
Scala
val re = "[\\x07\\x0A\\x0D\\x14]{7,}".r
[\\x07\\x0A\\x0D\\x14]{7,}
Kotlin
val regex = Regex("[\\x07\\x0A\\x0D\\x14]{7,}")
[\\x07\\x0A\\x0D\\x14]{7,}